conda/cli/main_run.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import json
+import os
+from os.path import isfile, join
+
+from ..base.context import context
+from ..common.compat import on_win
+from ..gateways.subprocess import subprocess_call
+
+# OLD conda-run help
+"""
```

Comment:
This is the old `conda run --help` pulled from the 3.x branch.  The old `conda run` for whatever reason assumed that there would only be a single executable per package, and so you would "run a package."